plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -0,0 +1,126 @@
+package org.jetbrains.dokka.base.translators.documentables
+
+import org.jetbrains.dokka.base.transformers.pages.comments.CommentsToContentConverter
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.Enum
+import org.jetbrains.dokka.model.Function
+import org.jetbrains.dokka.model.doc.TagWrapper
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.utilities.DokkaLogger
+
+open class DefaultPageCreator(
+    commentsToContentConverter: CommentsToContentConverter,
+    val logger: DokkaLogger
+) {
+    protected open val contentBuilder = PageContentBuilder(commentsToContentConverter, logger)
+
+    open fun pageForModule(m: Module): ModulePageNode =
+        ModulePageNode(m.name.ifEmpty { "root" }, contentForModule(m), m, m.packages.map { pageForPackage(it) })
```

Comment:
I think it should be something like `<root>`, because it can clash with actual package named `root`.